### PR TITLE
Added Search Code

### PR DESCRIPTION
**Pull request by antrromet**
@smadha @chrismattmann @MBoustani Guys, I've added the basic search code. Its fully functional and can be rolled out. A couple of things to note though.
1. As of now, I'm displaying only the `title` and `dc-title` keys in the search results. We will have to change this. Again some kind of option I guess as to what results to be seen in the search results. Maybe a popup or a config file, similar to the discussion happening [here](https://github.com/MBoustani/GeoParser/issues/55#issuecomment-204164973).
2. Also, we need some paging kind of mechanism where the user can see further search results. As of now we are fetching just the first 10 (by default 10 rows are fetched in solr).
We need to decide about the design of the search results. Lemme know what you guys think.

### CODE REVIEW

**Comment by chrismattmann** [comment]
LOVE this @antrromet

File: geoparser_app/static/js/app.js [hunk 1]
```diff
@@ -108,61 +108,72 @@ $(function() {
...
+			var docLink = layerToIndexMap[feature.get('layer')] + "/select?q=id:%22"+eval(popupData[1])+"%22&wt=json&indent=true"
+            var url = docLink;
+            var popup_content = '';
+            
+            
+            jQuery.ajax({
+                url: url,
+                data: '',
+                success:  function(res) {
+//                    console.log(JSON.stringify(res, null, 2));
```
Comment (chrismattmann) [inline geoparser_app/static/js/app.js:123]: remove since commented out

**Comment by antrromet** [comment]
Adding a screenshot for reference.
screen shot 2016-04-02 at 11 42 21 am

**Comment by smadha** [comment]
@antrromet I see some code which I have already merged in PR #61 
I just created a new branch [search](https://github.com/MBoustani/GeoParser/tree/search) from master. Can we work on this branch please?

Maybe you can close this PR and raise it again on new branch.

Thanks

**Comment by chrismattmann** [comment]
@smadha can you just merge this PR into the branch just gir pull right

**Comment by chrismattmann** [comment]
*git

**Comment by antrromet** [comment]
@smadha Lemme know if you are unable to do so. I will make another PR if needed.

**Comment by smadha** [comment]
@chrismattmann @antrromet I can try doing that. But as I said there is some code which I have already merged in PR #61. That's why I asked for new one to make it cleaner

But let me try merge this PR into the branch and I will update here

**Comment by smadha** [comment]
Merged via #64 in [GeoParser/tree/search](https://github.com/MBoustani/GeoParser/tree/search)

**Comment by MBoustani** [comment]
@antrromet looks great.

**Pull request closed without merging**